stacks_queues/stack.py

```diff
@@ -12,7 +12,8 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)
+        return None
```

Comment:
Small style tip: If a function always returns `None`, I'd recommend returning `None` implicitly by just not having any return value (functions in Python return `None` by default). 
```suggestion
```